fire_two.py

```
@@ -0,0 +1,102 @@
+import threading
+import requests
+import hmac
+import hashlib
+
+NGROK_URL = "https://3c0d-157-119-83-207.ngrok-free.app/civicrm/payment/ipn/civirazorpay?processor_id=7"
+WEBHOOK_SECRET = "LGS75qEToFNE2o6UIHjqFwvb"
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to: "WEBHOOK_SECRET"

(S105)